### PR TITLE
Added Tony to alums. #189

### PR DESCRIPTION
**Pull request by aaronjamesreynolds**

### CODE REVIEW

**Comment by albeanth** [comment]
@aaronjamesreynolds if this is all you were doing here, then 🚢 it.

Descriptions are great, even for the simple stuff!

**Comment by aaronjamesreynolds** [comment]
Yeah, I need to be better about that. Been playing in my own sandbox for too long.

**Comment by albeanth** [comment]
> Yeah, I need to be better about that. Been playing in my own sandbox for too long.

Even playing in your own sandbox... I super 100% recommend making detailed commit messages and stuff. There are few things as frustrating as trying to figure out what an old commit/MR was about... (anyway, I'll get off my soapbox now lol)

**Comment by aaronjamesreynolds** [comment]
> 
> 
> > Yeah, I need to be better about that. Been playing in my own sandbox for too long.
> 
> Even playing in your own sandbox... I super 100% recommend making detailed commit messages and stuff. There are few things as frustrating as trying to figure out what an old commit/MR was about... (anyway, I'll get off my soap box now lol)

You're right. It will surely save my bacon at some point. Make the sandbox more hospitable.

**Review by nickwhitman1993 — APPROVED** [review]
Looks good. Ship :ship: it and dip it, bruther.

**Comment by nickwhitman1993** [comment]
Wait, is Aaron T. not in our research group anymore?

**Comment by aaronjamesreynolds** [comment]
I'm pretty certain he is entirely within Dr. Camille Palmer's group now. I'll check.

**Comment by aaronjamesreynolds** [comment]
Confirmed.

File: index.html [hunk 1]
```diff
@@ -44,15 +44,9 @@ <h4>Core Design and Fuel Analysis</h4>
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/robertschickler" class="user-mention">@robertschickler</a>)</li>
 <li><a href="{{ site.url }}users/stewryan/">Ryan Stewart</a> (<a href="https://github.com/ryanstwrt" class="user-mention">@ryanstwrt</a>)</li>
-<li><a href="{{ site.url }}users/tamashia/">Aaron Tamashiro</a> (<a href="https://github.com/aarontama" class="user-mention">@aarontama</a>)</li>
 <li><a href="{{ site.url }}users/hillszach/">Zachary Hills</a> (<a href="https://github.com/zachthills" class="user-mention">@zachthills</a>)</li>
 
 
-</ul>
```
Comment (nickwhitman1993) [inline index.html:52]: @rtrp/admins We should've kept the `</ul>`. Check the current page to see what happened. I'll open an issue.

**Pull request merged**